src/main/java/net/openhft/chronicle/core/Jvm.java

```diff
@@ -69,8 +67,11 @@
 /**
  * Utility class to access information in the JVM.
  */
-public enum Jvm {
-    ; // none
+public final class Jvm {
```

Comment:
Why? I thought that the Enum trick was neat. Anyway, won't it cause any binary compatibility issues?

Reply:
This will remove some methods not intended (e.g. `values()`) This is the reason we should only use the Enum trick in internal classes. I have updated the compatibility percent after reviewing the flagged issues there.